src/pages/home/uploadSong/BasicSongDetails.tsx

```diff
@@ -293,7 +324,7 @@ const BasicSongDetails: FunctionComponent = () => {
                 : "default"
             }
           >
-            { values.isMinting ? "Next" : "Upload" }
+            { values.isMinting ? "Next" : isInEditMode ? "Save" : "Upload" }
```

Comment:
Is the intention that even if I don't modify any data on the `BasicSongDetails` page, when I click save, it will still "save", navigate me to the library page, and show a toast saying "Updated song information"?